scalding-beam/src/main/scala/com/twitter/scalding/beam_backend/BeamBackend.scala

```diff
@@ -1,21 +1,21 @@
 package com.twitter.scalding.beam_backend
 
-import com.twitter.scalding.dagon.{FunctionK, Memoize, Rule}
 import com.twitter.chill.KryoInstantiator
 import com.twitter.chill.config.ScalaMapConfig
 import com.twitter.scalding.Config
 import com.twitter.scalding.beam_backend.BeamOp.{CoGroupedOp, MergedBeamOp}
+import com.twitter.scalding.dagon.{FunctionK, Memoize, Rule}
 import com.twitter.scalding.serialization.KryoHadoop
+import com.twitter.scalding.typed.OptimizationRules._
 import com.twitter.scalding.typed._
+import com.twitter.scalding.typed.cascading_backend.CascadingExtensions.ConfigCascadingExtensions
 import com.twitter.scalding.typed.functions.{
   FilterKeysToFilter,
   FlatMapValuesToFlatMap,
   MapValuesToMap,
   ScaldingPriorityQueueMonoid
 }
 
-import com.twitter.scalding.typed.cascading_backend.CascadingExtensions.ConfigCascadingExtensions
```

Comment:
I'm glad if we don't need this, but I'm also surprised. I meant to remove any of these that were unneeded.

Reply:
This hasn't been removed...just moved up with the ordering of imports. I think we need this.